### DISCUSSION TITLE
problem with showing up my repos contribution on my profile repos

### DISCUSSION THREAD

**Discussion by franzhd** [community/community#27282] [Other Feature Feedback, Questions, & Ideas]
Hello everyone, 
i am a team member and owner of the https://github.com/AML-Project-VG group with two friends of mine. I wanted to show the repo that we create and developed, project_vg, on my profile's main page, but when i serch for it the "customize your pins" window, it did not show up. I have the same settings of my friends, they can do that while i can't. what is happening? how can i solve this problem? 
thanks for your attention, 
wish your help :D

**Comment by jfullstackdev** [comment]
maybe they had forked the repo then pinned it, like mine 

![image](https://user-images.githubusercontent.com/47092464/181236283-b274e888-65cd-4b0b-bb31-71cebd82c3f1.png)

**Comment by radar-blip** [comment]
There are 2 project_vg. Try pinning the one that isnt forked. or both.

**Comment by franzhd** [comment]
the problem is that i cannot see both of them in "my repo" or inside  the window that allow me to pin stuff on my main page, there isn't the name of that repo. If i fork it i can show it, otherwise i cannot.

![image](https://user-images.githubusercontent.com/58605124/181880428-97cf85d3-1a68-4ae2-97d5-1c382d13d050.png)

**Comment by jfullstackdev** [comment]
because first, you don't own the repo, and second, you did not fork the repo 

![image](https://user-images.githubusercontent.com/47092464/181904333-39b0e756-b595-45ee-9891-0524bdc9765f.png)

as I mentioned, fork it first, then pin it

there is also an external service to pin that on your README

paste this on that README of your profile

```
[![Readme Card](https://github-readme-stats.vercel.app/api/pin/?username=AML-Project-VG&repo=project_vg&theme=tokyonight)](https://github.com/AML-Project-VG/project_vg)
```

and you'll see this 

[![Readme Card](https://github-readme-stats.vercel.app/api/pin/?username=AML-Project-VG&repo=project_vg&theme=tokyonight)](https://github.com/AML-Project-VG/project_vg)

more info about that external service

https://github.com/anuraghazra/github-readme-stats

**Reply by freealise** [reply]
This is really strange - I have an opposite problem. There's a repo among my suggested pins that I contributed to many years ago and keep wondering how to remove it as I am not interested in it whatsoever. Is there a way?

**Reply by jfullstackdev** [reply]
you might have the fork of it, simply delete the fork and that should be gone in the list of repos that you can pin

![image](https://user-images.githubusercontent.com/47092464/182744133-382eb26b-be45-44eb-b697-855653de96aa.png)

**Reply by freealise** [reply]
No, it's not among my forks (the last one on the list).

![Screenshot 2022-08-04 at 04 41 44](https://user-images.githubusercontent.com/5275747/182744920-7b55daab-d82d-4440-aad1-099a62a9022d.png)

**Reply by jfullstackdev** [reply]
![image](https://user-images.githubusercontent.com/47092464/182746720-7d41282f-1aea-4881-b8f3-d85385accff0.png)

https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/pinning-items-to-your-profile

not very sure whether this is possible in your situation, this will only apply, I think if you are added directly

https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-access-to-your-personal-repositories/removing-yourself-from-a-collaborators-repository

**Reply by freealise** [reply]
This probably does not apply to past (and closed) issues and pull requests, since it's not there either.

![Screenshot 2022-08-04 at 15 11 06](https://user-images.githubusercontent.com/5275747/182844437-6078b29b-35b1-4a9a-b8c9-46da47780d2f.png)

**Reply by franzhd** [reply]
how is it possible that i don't own the repository ? i am one of the admins of the organization and contributor to the repo. all my peers have same roles and settings that i have. how is it possible that they can show it on their profile without forking it ?
